### DISCUSSION TITLE
Is there a way to specify a raw string (no escapes)?

### DISCUSSION THREAD

**Discussion by dropwhile** [gleam-lang/gleam#2742] [Questions & support] [answered]
When working with regex, additional escapes can be very confusing. In other languages, “raw strings” are often useful for this.

Does gleam support raw strings?
I looked around but didn’t find any documentation on it.. even the examples in the stdlib regex docs ([example](https://hexdocs.pm/gleam_stdlib/gleam/regex.html#scan)) show double backslash to escape the escapes.

**Comment by lpil** [comment, marked as answer]
Not currently, no.
